Kitodo-DataManagement/src/main/resources/db/migration/V2_138__Switch_to_Hibernate-Search.sql

```diff
@@ -21,3 +21,8 @@ ALTER TABLE ruleset DROP COLUMN indexAction;
 ALTER TABLE task DROP COLUMN indexAction;
 ALTER TABLE template DROP COLUMN indexAction;
 ALTER TABLE workflow DROP COLUMN indexAction;
+
+--
+-- Add index for better task search
+
+CREATE INDEX idx_task_filtering_title ON task (process_id, typeAutomatic, processingStatus, title);
```

Comment:
There are some on-going discussion about the indices which should be used or not. See https://github.com/matthias-ronge/kitodo-production/pull/4#issuecomment-2850600905 . This index is not bad and speeds up the search already but there is at least one or up to three others as a hopefully better replacement for this index.